packages/atomic/src/components/result-template-components/result-template-decorators.ts

```diff
@@ -1,25 +1,35 @@
+import {Result} from '@coveo/headless';
 import {ComponentInterface, getElement} from '@stencil/core';
+import {buildCustomEvent} from '../../utils/event-utils';
+
+export class MissingResultParentError extends Error {
```

Comment:
What is the benefit of making this a class versus before ?

Just for code clarity sake ?

Reply:
Yes